Assets/HoloToolkit/Input/Scripts/GestureManager.cs

```diff
@@ -218,27 +256,36 @@ void LateUpdate()
                 newFocusedObject = OverrideFocusedObject;
             }
 
-            if (FocusedObject != newFocusedObject)
-            {
-                // If the currently focused object doesn't match the old focused object, cancel the current gesture.
-                // Start looking for new gestures.  This is to prevent applying gestures from one hologram to another.
-                gestureRecognizer.CancelGestures();
-                FocusedObject = newFocusedObject;
-                gestureRecognizer.StartCapturingGestures();
-            }
+            bool focusedChanged = FocusedObject != newFocusedObject;
 
 #if UNITY_EDITOR
             if (Input.GetMouseButtonDown(1) || Input.GetKeyDown(EditorSelectKey))
```

Comment:
Would we want to check if focus has changed here so we don't tap on the object if it's not our focused object?